core/src/main/java/com/capitalone/dashboard/util/DateUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.capitalone.dashboard.util;
+
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * User: Alan
+ * Email: alan@hialan.com
+ * Date: 16:50 4/13/16
+ */
+public class DateUtils {
```

Comment:
if a class only has a static method, should just make the constructor to be private